grpc/src/test/java/com/linecorp/armeria/server/grpc/GrpcDocServicePluginTest.java

```diff
@@ -211,9 +211,9 @@ public void newEnumInfo() throws Exception {
         final EnumInfo enumInfo = generator.newEnumInfo(CompressionType.getDescriptor());
         assertThat(enumInfo).isEqualTo(new EnumInfo(
                 "armeria.grpc.testing.CompressionType",
-                ImmutableList.of(new EnumValueInfo("NONE"),
-                                 new EnumValueInfo("GZIP"),
-                                 new EnumValueInfo("DEFLATE"))));
+                ImmutableList.of(new EnumValueInfo("NONE", 0),
+                                 new EnumValueInfo("GZIP", 1),
```

Comment:
Can you change the proto file so these numbers aren't in sequence (e.g., change DEFLATE to `= 3`. Then this test verifies that we correctly display the proto value, not enum ordinal.

Reply:
Yes that's a good idea